.pre-commit-config.yaml

```diff
@@ -241,7 +241,7 @@ repos:
       - id: setup-order
         name: Check order of dependencies in setup.py
         language: python
-        files: ^setup.py$
+        files: ^setup.py$|^setup.cfg$
```

Comment:
Don't need this anymore do we?

Reply:
We do. There is a check that checks sequence in setup.cfg as well :)

Reply:
You've updated it when you moved the "install_requires".

Reply:
But the comment + description of this commit + the files spec were not updated then. I modified the description now to be clear that 'setup-order' concerns both .py and .cfg